CSS/main.css

```diff
@@ -433,3 +440,242 @@ main {
   background-repeat: no-repeat;
   background-position: center;
 }
+
+/* Hiding some desktop features in the mobile version */
+@media screen and (max-width: 990px) {
+  header nav .nav-links,
+  .desktop-image,
+  .hide-for-mobile {
+    display: none;
+  }
+}
+
+/* Mobile and Medium */
+@media screen and (min-width: 600px) and (max-width: 990px) {
+  .headline {
+    padding: 6.5rem 5rem 5.59375rem 5rem;
+  }
+
+  .about-myself {
+    justify-content: center;
+    flex-direction: row;
+    padding: 4.375rem 4.375rem 4.1875rem 4.375rem;
+  }
+
+  .about-myself .top-container {
+    width: 60.3%;
+  }
+
+  .skill-lists {
+    width: 40.7%;
+  }
+
+  .contact-sect h2,
+  .contact-sect p {
+    max-width: 477px;
+  }
+
+  .contact-form {
+    max-width: 350px;
+  }
+}
+
+/* BIG SCREENS */
+@media screen  and (min-width: 991px) {
```

Comment:
- [ ] Please pick a value from `992 - 1100px` , the present value is too low 🙏

Reply:
Thanks for your review. I have increased the value to 992px as you suggested